VERSION

```diff
@@ -1 +1 @@
-58.0.3029.110
+59.0.3071.86
```

Comment:
59.0.3071.104 just came out with several security fixes, we should upgrade to that version.

https://chromereleases.googleblog.com/2017/06/stable-channel-update-for-desktop_15.html